### ISSUE TITLE
Classification store field only shows one group since pimcore 6.8.12

### ISSUE THREAD

**Issue by juckerf** [pimcore/pimcore#8865]
# Bug Report

### Expected behavior
When an object has a classification store field with multiple active groups we expect that all those groups are shown in the object edit view.
![image](https://user-images.githubusercontent.com/7573298/115738010-f10f6900-a38c-11eb-9946-f80969b046a6.png)

### Actual behavior
Only the group with id = 1 is display in the object edit view (see screenshot)
![image](https://user-images.githubusercontent.com/7573298/115737899-db9a3f00-a38c-11eb-9799-270b40b7306c.png)

### Steps to reproduce (Demo)
* Create classification store "test"
* Define classification store field on car class referencing the store "test"
* create multiple groups and attributes in the store "test"
* assign multiple groups to any car
* save and reload
* only one group is diplayed

I think the bug was introduced in 6.8.12 with the following line:
https://github.com/pimcore/pimcore/blob/v6.8.12/models/DataObject/ClassDefinition/Data/Classificationstore.php#L1231

This leads to a MySQL-Query as shown (note the single quotes around the list of IDs):
![Selection_086](https://user-images.githubusercontent.com/7573298/115739685-6b8cb880-a38e-11eb-878d-e04bc8be6264.png)

Back in 6.8.11 it wasn't a prepared statement but a sketchy string concatenation (https://github.com/pimcore/pimcore/blob/v6.8.10/models/DataObject/ClassDefinition/Data/Classificationstore.php#L1251). The MySQL-Query looked like this: 
![Selection_087](https://user-images.githubusercontent.com/7573298/115739798-82cba600-a38e-11eb-9ae3-f5c0d76e354f.png)

### Possible solution
something like this:
```php
$groupList->setCondition('`id` in (' . implode(',', array_map(function ($id) { return '?'; }, $filteredGroupIds)) . ')', $filteredGroupIds);
```
=> produces a query with the same amount of bind variables like IDs in `$filteredGroupIds` and provides $filteredGroupIds as parameters

**Comment by weisswurstkanone** [comment]
Hello @juckerf , could you prepare a PR please ? Thanks a lot.

**Comment by NiklasBr** [comment]
Looks like my changes introduced the issue, I have a PR to fix it with the code @juckerf suggested.

**Comment by juckerf** [comment]
Thanks for the PR @NiklasBr !

**Comment by brusch** [comment]
Fixed by #8872
